packages/valory/skills/decision_maker_abci/utils/scaling.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2024 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+
+"""This package contains helpers for scaling operations."""
+
+
+from typing import List, Tuple
+
+
+def min_max(li: List[float]) -> Tuple[float, float]:
+    """Get the min and max of a list."""
+    if not li:
+        raise ValueError("The list is empty.")
+
+    min_ = max_ = li[0]
+
+    for num in li[1:]:
+        if num < min_:
+            min_ = num
+        elif num > max_:
+            max_ = num
+
+    return min_, max_
+
+
+def scale_value(
+    value: float,
+    min_max_bounds: tuple[float, float],
+    scale_bounds: tuple[float, float] = (0, 1),
+) -> float:
+    """Perform min-max scaling on a value."""
+    min_, max_ = min_max_bounds
+    current_range = max_ - min_
+    # normalize between 0-1
+    std = (value - min_) / current_range
```

Comment:
just after this line you can do `return std`

Reply:
https://github.com/valory-xyz/trader/pull/296#discussion_r1683018968